grc/workflows/cpp_hb_nogui/flow_graph_hb_nogui.cpp.mako

```diff
@@ -0,0 +1,126 @@
+<%def name="doubleindent(code)">${ '\n        '.join(str(code).splitlines()) }</%def>\
+/********************
+GNU Radio C++ Flow Graph Source File
+
+Title: ${title}
+% if flow_graph.get_option('author'):
+Author: ${flow_graph.get_option('author')}
+% endif
+% if flow_graph.get_option('description'):
+Description: ${flow_graph.get_option('description')}
+% endif
+GNU Radio version: ${config.version}
+********************/
+
+#include "${flow_graph.get_option('id')}.hpp"
+% if flow_graph.get_option('realtime_scheduling'):
+#include <gnuradio/realtime.h>
+% endif
+
+% if parameters:
+
+namespace po = boost::program_options;
+
+% endif
+using namespace gr;
+
+<%
+class_name = flow_graph.get_option('id') + ('_' if flow_graph.get_option('id') == 'top_block' else '')
+
+param_str = ", ".join((param.vtype + " " + param.name) for param in parameters)
+param_str_without_types = ", ".join(param.name for param in parameters)
+initializer_str = ",\n  ".join((param.name + "(" + param.name + ")") for param in parameters)
+
+if len(initializer_str) > 0:
+  initializer_str = '\n: ' + initializer_str
+%>\
+
+${class_name}::${class_name} (${param_str}) ${initializer_str} {
+
+% if flow_graph.get_option('thread_safe_setters'):
+## self._lock = threading.RLock()
+% endif
+    this->tb = gr::make_top_block("${title}");
+
+% if blocks:
+// Blocks:
+% for blk, blk_make, declarations in blocks:
+        ${doubleindent(blk_make)}
+##      % if 'alias' in blk.params and blk.params['alias'].get_evaluated():
+##      ${blk.name}.set_block_alias("${blk.params['alias'].get_evaluated()}")
+##      % endif
+##      % if 'affinity' in blk.params and blk.params['affinity'].get_evaluated():
+##      ${blk.name}.set_processor_affinity("${blk.params['affinity'].get_evaluated()}")
+##      % endif
+##      % if len(blk.sources) > 0 and 'minoutbuf' in blk.params and int(blk.params['minoutbuf'].get_evaluated()) > 0:
+##      ${blk.name}.set_min_output_buffer(${blk.params['minoutbuf'].get_evaluated()})
+##      % endif
+##      % if len(blk.sources) > 0 and 'maxoutbuf' in blk.params and int(blk.params['maxoutbuf'].get_evaluated()) > 0:
+##      ${blk.name}.set_max_output_buffer(${blk.params['maxoutbuf'].get_evaluated()})
+##      % endif
```

Comment:
I'd welcome removal or a comment explaining why it's commented out?

Reply:
@ZakyHermawan @haakov any ideas? I'll remove the comments otherwise.

Reply:
@marcusmueller ah OK. This is simply exactly as it is on `main` branch, in the corresponding files. If we change this, then that change should happen in a different commit.